### DISCUSSION TITLE
Como ingresar al correo de innovación que nos suministratron

### DISCUSSION THREAD

**Discussion by Linamar126** [Launch-X-Latam/MisionFrontEnd#57] [Q&A]
Buen día! Mientras realicé el Onbording pude ingresar al correo de innovación por medio del mismo Teams, ahora ya no me aparece la opción. Alguno sabe como puedo volver a ingresar a dicho correo? pues creo que a través de ese nos estarán llegando los comunicados de innovación.

**Comment by ghost** [comment]
Hola buen día por Microsoft Outlook  inicias sesión con el correo que te dieron  y te van a llegar los mensajes ![Screenshot_20220928_082652_com.microsoft.office.outlook.jpg](https://user-images.githubusercontent.com/114112929/192791707-f87675e7-3a99-4dc7-8da7-27216da57b78.jpg)

**Comment by doguedogue** [comment]
El correo de innovaccion te sirve para entrar a Teams, puedes acceder a office.com --> luego a outlook. 

Los avisos llegan por Teams y si pueden llegar a tu correo de innovaccion o al correo con el que iniciaste toda la aventura.

Nota: Si tienes otra cuenta de microsoft que uses outlook trata de cerrar sesion o usa una ventana de incognito.

**Comment by JorgeDevmm** [comment]
> ¡Buen día! Mientras realicé el Onbording pude ingresar al correo de innovación por medio del mismo Teams, ahora ya no me aparece la opción. Alguno sabe como puedo volver a ingresar a dicho correo? pues creo que a través de esos nos están llegando los comunicados de innovación.

solo ingresa de nuevo por Outlook y te debería mostrar los correos, también se te habilita la opción de aplicación como teams ahi mismo

**Comment by gjordanra** [comment]
Si entras al teams via web, arriba a la izquierda en el cuadrado de puntitos das clic, te muestra los otros programas de microsoft que puedes usar en tu cuenta de innovaccion, ahi ves el icono del outlook, le das clic
![image](https://user-images.githubusercontent.com/4700976/192842961-265c8f9c-c0e4-4ebc-957a-e6505d112de2.png)

**Comment by Linamar126** [comment]
Gracias a todos por sus respuestas! con sus soluciones logré ingresar de nuevo al correo!!!
